### PR TITLE
Widget/slider

### PR DESCRIPTION
**Pull request by FrozenKiwi**

### CODE REVIEW

File: apps/site-app/package.json [hunk 1]
```diff
@@ -71,10 +71,8 @@
     "react-helmet": "6.0.0-beta",
     "react-measure": "^2.3.0",
     "react-semantic-ui-range": "^0.7.0",
-    "react-transition-group": "^4.3.0",
     "reselect": "4.0.0",
-    "sanitize.css": "10.0.0",
-    "ts-clipboard": "^1.0.17"
```
Comment (FrozenKiwi) [inline apps/site-app/package.json:77]: noticed a lot of unused packages here. They gone now...

File: apps/site-landing/lang/en.json [hunk 2]
```diff
@@ -341,10 +341,6 @@
     "defaultMessage": "Success: check your emails",
     "description": "site.subscribe.success: Message we give a user when the subscription is a success"
   },
-  "tk3UgV": {
```
Comment (FrozenKiwi) [inline apps/site-landing/lang/en.json:344]: text should not be transalted

File: apps/site-landing/src/components/RangeFieldAndScale/index.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <FormattedNumber
+            value={value}
+            currency={props.currency}
+            currencyDisplay={"narrowSymbol"}
+            style={props.scaleType}
+            minimumFractionDigits={0}
+            maximumFractionDigits={0}
+           />
+        </div>
+        <Range
```
Comment (FrozenKiwi) [inline apps/site-landing/src/components/RangeFieldAndScale/index.tsx:47]: I couldn't get the in-build one to color the track non-uniformly, so installed a new version to do it.

File: apps/site-landing/src/containers/Compare/FormCompare/index.tsx [hunk 4]
```diff
@@ -26,55 +23,31 @@ const translations = defineMessages({
...
+    <div id={styles.variablesContainer}>
+
+      <RangeFieldAndScale
+        label={translations.startingValue}
+        scaleType="currency"
+        currency="CAD"
+        maximum={1000}
+        step={1}
+        onChange={onChange}
+      />
```
Comment (FrozenKiwi) [inline apps/site-landing/src/containers/Compare/FormCompare/index.tsx:39]: This file isn't the topic of review.  Updated to compile, doesn't matter if it 'works'

File: libs/site-semantic-theme/src/semantic/thecoin/elements/input.overrides [hunk 5]
```diff
@@ -8,12 +8,12 @@
 // definition could override this one
 .ui.input {
 
-  & > input {
+  & > input[type="text"] {
     font-weight: @bold;
 
     &:hover {
        border-color: @hoverBorderColor;
-       box-shadow: @hoverBoxShadow !important;
```
Comment (FrozenKiwi) [inline libs/site-semantic-theme/src/semantic/thecoin/elements/input.overrides:16]: important overrode the shadows on our range input, so had to apply it a different way

File: apps/site-landing/lang/en.json (hunk 2 above)
Comment (MarieTheCoin) [reply apps/site-landing/lang/en.json:344]: Transalted: for a second I thought I was learning a new word today

File: apps/site-landing/src/components/RangeFieldAndScale/Slider.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      default: 'graphColor',
+      values: [
+        { name: 'graphColor', value: '#138175' },
+      ],
+    },
+  },
+  argTypes: { onChange: { action: 'onChange' } },
+  args: {
+    label: "Label",
+    currency: "CAD" as "CAD",
```
Comment (MarieTheCoin) [inline apps/site-landing/src/components/RangeFieldAndScale/Slider.stories.tsx:20]: That's a surprising line. What is it doing?

**Review by MarieTheCoin — APPROVED** [review]

File: apps/site-landing/src/components/RangeFieldAndScale/Slider.stories.tsx (hunk 6 above)
Comment (FrozenKiwi) [reply apps/site-landing/src/components/RangeFieldAndScale/Slider.stories.tsx:20]: ha!  Yeah, its kinda WTF :-).

Our slider displays a currency. One day we might have multiple currencies, so it accepts an argument (which kind of currency).  But we want to type-check the type of currency, so the only one accepted is type "CAD".  But when we put the argument here of "CAD", TS thinks it's of type string unless I explicitly override it with type "CAD".

If I put "CAD" directly as argument to component, I wouldn't need it, it's just cause I let it come in as an argument.

All technically correcter, but prolly more work than it's worth.

**Pull request merged**